websocketOutportDriver/payloadParser.go

```diff
@@ -94,3 +94,8 @@ func padUint32ByteSlice(initial []byte) []byte {
 	padding := bytes.Repeat([]byte{0}, 4)
 	return append(padding, initial...)
 }
+
+// IsInterfaceNil -
+func (wpp *websocketPayloadParser) IsInterfaceNil() bool {
```

Comment:
this can also be tested in the payloadParser_test.go where you test the constructor and replace
```GO
assert.NotNil(t, instance)
```
with
```GO
assert.False(t, check.IfNil(instance))
```

Reply:
Right, applied suggestion